src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -19,6 +19,7 @@ public class SearchRequest {
     private String cropMarker;
     private String highlightPreTag;
     private String highlightPostTag;
+    private String matchingStrategy;
```

Comment:
What happens if you change this to `MatchingStrategy` instead of String?

Reply:
I could but it forces the user to use the enum, for me it's better if we let the choice to the user to use both way no?